core/src/main/java/nl/nn/adapterframework/pipes/IncreaseIntegerPipe.java

```diff
@@ -52,6 +57,20 @@ public PipeRunResult doPipe(Message message, IPipeLineSession session) throws Pi
 
 		String sessionKeyString = (String) session.get(sessionKey);
 		Integer sessionKeyInteger = Integer.valueOf(sessionKeyString);
+
+		ParameterValueList pvl = null;
+		if (getParameterList() != null) {
+			try {
+				pvl = getParameterList().getValues(message, session);
+			} catch (ParameterException e) {
+				throw new PipeRunException(this, getLogPrefix(session) + "exception extracting parameters", e);
+			}
+		}
+		ParameterValue pv = pvl.getParameterValue(PARAMETER_INCREMENT);
+		if(pv != null && pv.getValue() != null) {
+			increment = pv.asIntegerValue(increment);
+		}
```

Comment:
The != check is also present in `asIntegerValue`.
Right now `pv.asIntegerValue(increment);` will never use the argument `increment`.
I think it's better if you either remove the != null check, or replace this argument with 0 or something?